pkg/package.go

```diff
@@ -11,7 +11,7 @@ import (
 	"golang.org/x/xerrors"
 )
 
-type PackageInfo struct {
+type commonPackageInfo struct {
```

Comment:
I would like to avoid creating a new struct just for testing as much as possible.

Reply:
Do you have an alternative then to having to recreate the test data every time a field is added? Some of the test data isn't even possible to recreate, given that CentOS 5 (and 6 I think?) repos do not even exist anymore.

In fact, I think the 'testdata/centos5-plain/Packages' is a v4 RPM db when it should be v3. So I'm not even sure that it's testing the right thing.

Reply:
> Do you have an alternative then to having to recreate the test data every time a field is added? Some of the test data isn't even possible to recreate, given that CentOS 5 (and 6 I think?) repos do not even exist anymore.
> 
> In fact, I think the 'testdata/centos5-plain/Packages' is a v4 RPM db when it should be v3. So I'm not even sure that it's testing the right thing.

As I said in my other comment, I think I have a better direction to go. 

But also, do you think there is still utility in the old version tests? CentOS < 7?